server/src/testFixtures/java/org/elasticsearch/test/IntegTestCase.java

```diff
@@ -1421,7 +1421,9 @@ public void assertNoTasksAreLeftOpen() throws Exception {
                     throw new RuntimeException(ex);
                 }
             }
-            throw new AssertionError(errorMessageBuilder.toString(), e);
+            String errMsg = errorMessageBuilder.toString();
+            LOGGER.error("Open jobs={}", errMsg);
```

Comment:
Was for debugging but I'm tempted to keep it as it helps associating a leak to the right output when running tests with `Dtests.iters=20` or similar.